hoon/hoon-school/atoms-auras-and-simple-cell-types.md

```diff
@@ -109,6 +109,8 @@ Aura         Meaning                        Example Literal Syntax
 
 Some of these auras nest under others.  For example, `@u` is for all unsigned auras.  But there are other, more specific auras; `@ub` for unsigned binary numbers, `@ux` for unsigned hexadecimal numbers, etc.
 
+For a more complete list of auras, see [Auras](@/docs/hoon/reference/auras.md).
+
```

Comment:
I would say it is more *discussion* of auras, since the list itself is a copy of the code block above and thereby inherently no more or less complete.

Reply:
It's not a copy - there are auras listed in this PR that are not in the Hoon school notes. It is an extension and thus more complete.

I did not edit the Hoon school notes to include the additional auras in this PR for pedagogical reasons - when someone is learning something new, its not necessarily a good idea to include all associated complexity right off the bat. But I would be willing to consider that that might not apply here, or that some of these additional auras should be listed in the Hoon school notes.